### ISSUE TITLE
Missing Accessibility Hazards Metadata in examples

### ISSUE THREAD

**Issue by clapierre** [w3c/epub-specs#2777]
### Section

3.1

### Describe the problem

There needs to be accessibilityHazard of "none" for books with no sound/video, and others where there are flashing or sound hazards we need to call out specifically noXYZHazard for the others.
